src/app/components/PageLayout/ScrollingCard.tsx

```diff
@@ -0,0 +1,20 @@
+import Card from '@mui/material/Card'
+import { FC, useEffect, useRef } from 'react'
+import { useLocation } from 'react-router-dom'
+
+export const ScrollingCard: FC<Parameters<typeof Card>[0]> = props => {
+  const { id } = props
+  const { hash } = useLocation()
+  const cardRef = useRef<HTMLDivElement>(null)
+  const card = cardRef.current
+
+  useEffect(() => {
+    if (id && hash === `#${id}` && card) {
+      card.scrollIntoView({
+        behavior: 'smooth',
```

Comment:
This line might be unnecessary after https://github.com/oasisprotocol/explorer/pull/251 is merged.

Reply:
Indeed. Removed.